src/Tribe/Importer/Column_Mapper.php

```diff
@@ -71,27 +71,29 @@ private function get_event_column_names() {
 	}
 
 	private function get_venue_column_names() {
-		return array(
-			'venue_name'     => esc_html__( 'Venue Name', 'the-events-calendar' ),
-			'venue_country'  => esc_html__( 'Venue Country', 'the-events-calendar' ),
-			'venue_address'  => esc_html__( 'Venue Address', 'the-events-calendar' ),
-			'venue_address2' => esc_html__( 'Venue Address 2', 'the-events-calendar' ),
-			'venue_city'     => esc_html__( 'Venue City', 'the-events-calendar' ),
-			'venue_state'    => esc_html__( 'Venue State/Province', 'the-events-calendar' ),
-			'venue_zip'      => esc_html__( 'Venue Zip', 'the-events-calendar' ),
-			'venue_phone'    => esc_html__( 'Venue Phone', 'the-events-calendar' ),
-			'venue_url'      => esc_html__( 'Venue Website', 'the-events-calendar' ),
-			'featured_image' => esc_html__( 'Venue Featured Image', 'the-events-calendar' ),
-		);
+		return apply_filters( 'tribe_events_importer_venue_column_names', array(
+			'venue_name'        => esc_html__( 'Venue Name', 'the-events-calendar' ),
+			'venue_description' => esc_html__( 'Venue Description', 'the-events-calendar' ),
+			'venue_country'     => esc_html__( 'Venue Country', 'the-events-calendar' ),
+			'venue_address'     => esc_html__( 'Venue Address', 'the-events-calendar' ),
+			'venue_address2'    => esc_html__( 'Venue Address 2', 'the-events-calendar' ),
+			'venue_city'        => esc_html__( 'Venue City', 'the-events-calendar' ),
+			'venue_state'       => esc_html__( 'Venue State/Province', 'the-events-calendar' ),
+			'venue_zip'         => esc_html__( 'Venue Zip', 'the-events-calendar' ),
+			'venue_phone'       => esc_html__( 'Venue Phone', 'the-events-calendar' ),
+			'venue_url'         => esc_html__( 'Venue Website', 'the-events-calendar' ),
+			'featured_image'    => esc_html__( 'Venue Featured Image', 'the-events-calendar' ),
+		) );
 	}
 
 	private function get_organizer_column_names() {
 		return array(
```

Comment:
Optionally, could be nice to make this filterable (like with `get_venue_column_names()`), too.